### DISCUSSION TITLE
IPadOS 18 beta

### DISCUSSION THREAD

**Discussion by ash34** [isontheline/pro.webssh.net#1170] [General]
I wiped my iPad and installed the iPadOS 18 beta. First thing I did was install TestFlight then WebSSH. Has anybody else tried this combination yet?

First thing I see is that I’m restricted to only one ssh connection. Maybe this is a TestFlight thing and I lost the beta member status?

The reason for the iPad wiping is to try to narrow down a Wi-Fi issue. I have to disconnect and reconnect every time I pick up the iPad. It’s driving me crazy. Not convinced it’s a software issue either.

**Comment by isontheline** [comment]
> Maybe this is a TestFlight thing and I lost the beta member status?

Yes you should restore over the App Store first in order to retrieve the "receipt".
If not please come back to me through email : team [AT) webssh {DOT] net

> Has anybody else tried this combination yet?

Only tried on the simulator not any real device yet

**Comment by ash34** [comment]
I managed to solve it by enabling iCloud in the WebSSH settings. 

No problems found with the app on iPadOS 18 so far.

**Reply by isontheline** [reply]
> I managed to solve it by enabling iCloud in the WebSSH settings.

Perfect!

> No problems found with the app on iPadOS 18 so far.

Big thanks for your test 🙏
